app/src/main/res/xml/preferences.xml

```diff
@@ -30,7 +31,9 @@
 
     <androidx.preference.PreferenceCategory
         android:title="@string/preferences_category_general"
-        app:iconSpaceReserved="false">
+        app:iconSpaceReserved="false"
+        tools:ignore="PrivateResource"
```

Comment:
Normally, this is a super scary lint warning to suppress.  This is saying that the Android component doesn't consider that field a part of its API, which means they could push an update at any time that makes this code not work.

In _*this*_ case it looks like a false positive though.  It seems to be linting for the filename without regard for the namespace.  If you change the name of `preference_category.xml` those linting errors will go away, and we won't need this suppression.

Btw, it's a good policy to either fix or suppress all lint errors in your PR (and personally if I have to suppress, I try to write a comment explaining why).